Java/fclUserManagement/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1,11 @@
+F:\Developments\java 2020\fclUserManagement\src\main\java\com\fcl\users\NewUserRestService.java
```

Comment:
Why point to a F:\ path?
Co developer may/may not have the same drive setup.

Suggestion to use configurable env variable if possible